File: web/src/types/evaluation.ts
export interface EvaluationDetail {
  projectId: number
  projectNo: string
  title: string
  stage: string
  status: string
  statusLabel: string
  currentNode: string
  enterpriseName?: string
  precheckRemark?: string
  conditionResult?: string
  conditionRemark?: string
  rectifyNotice?: string
  resourceRemark?: string
  resourceRequirement?: string
  feasibilityResult?: string
  feasibilityRemark?: string
  conclusion?: string
  conclusionOpinion?: string
  feedbackContent?: string
  materials: Array<{ fileUrl?: string; fileName?: string; materialType?: string }>
  steps: Array<{ node: string; status: string }>
  logs: Array<{ remark: string; time: string; fromStatus: string; toStatus: string }>
}

export interface EvaluationRequest {
  remark?: string
  notice?: string
  requirement?: string
  conclusion?: string
  opinion?: string
  content?: string
  passed?: boolean
  materials?: Array<{ fileUrl?: string; fileName?: string; materialType?: string }>
}

File: server/src/main/java/com/gzpprod/center/module/evaluation/entity/Evaluation.java
package com.gzpprod.center.module.evaluation.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.time.LocalDateTime;

@Data
@TableName("evaluation")
public class Evaluation {

    @TableId(type = IdType.AUTO)
    private Long id;
    private Long projectId;
    private String precheckRemark;
    private String conditionResult;
    private String conditionRemark;
    private String rectifyNotice;
    private String resourceRemark;
    private String resourceRequirement;
    private String feasibilityResult;
    private String feasibilityRemark;
    private String conclusion;
    private String conclusionOpinion;
    private String feedbackContent;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}
